lib/utils/device/deploy.ts

```diff
@@ -429,6 +429,7 @@ export async function rebuildSingleTask(
 		convertEol: opts.convertEol,
 		multiDockerignore: opts.multiDockerignore,
 		nogitignore: opts.nogitignore,
+		buildargs: opts.buildargs,
```

Comment:
This addresses a [previous comment](https://github.com/balena-io/balena-cli/pull/2110#issuecomment-781722873) of mine:

> I think (not sure) we may need to check whether the function `rebuildSingleTask()`, which is called by livepush when certain changes are made to the filesystem, also needs to keep track of build arguments (we wouldn't want build arg flags to be "lost" when livepush rebuilds user containers):
https://github.com/balena-io/balena-cli/blob/v12.40.0/lib/utils/device/deploy.ts#L425
> Maybe we should revisit all calls to tarDirectory.